Update CONTRIBUTING.md (#4439)
the-turing-way/the-turing-way: CONTRIBUTING.md

- [Slack Channel](https://join.slack.com/t/theturingway/shared_invite/zt-2v7euwuo7-BYstHdKuTNd1ce0puDtBxA)
  - For structured discussion and sustained engagement with the community members.
  - We will also provide notifications on upcoming events and share useful resources on Slack.

> - Contact the Research Community Manager and other members of _The Turing Way_ project delivery team by emailing [theturingway@gmail.com](mailto:theturingway@gmail.com).

Reviewer: The sentence needs changing - maybe Contact the Research Community Management working group and the Steering group by emailing ...

Reviewer 2: ```suggestion
- Contact the Research Community Management working group and the Steering group by emailing [theturingway@gmail.com](mailto:theturingway@gmail.com).
```
